packages/analytics/lib/index.js

```diff
@@ -84,7 +84,7 @@ export {
   logViewSearchResults,
   setDefaultEventParameters,
   initiateOnDeviceConversionMeasurementWithEmailAddress,
-} from '../modular/index';
+} from './modular/index';
```

Comment:
amazing that worked before! How did that work in fact, without testing indicating some sort of failure 🤔

Reply:
I moved the file from outside `lib` to inside it to follow how we are doing it in other places. So just a small change in the import :)

Reply:
Ah, I didn't catch the file move, that makes sense